pages.ko/common/bw.md

```diff
@@ -0,0 +1,24 @@
+# bw
+
+> Bitwarden 보관함에 접속과 관리를 위한 CLI.
+> 더 많은 정보: <https://help.bitwarden.com/article/cli/>.
+
+- Bitwarden 사용자 계정 로그인:
+
+`bw login`
+
+- Log out of a Bitwarden 사용자 계정 로그아웃:
```

Comment:
This isn't fully translated?

Reply:
oh thanks! I forgot when I was translating next to the original text.